android/fastlane/Fastfile

```diff
@@ -25,6 +25,14 @@ platform :android do
     gradle(task: "test")
   end
 
+  desc "Build the app in release mode"
```

Comment:
looks like description (release mode) doesn't reflect the actual build (test/debug)